apps/gallery/components/wui-gallery-div/index.ts

```diff
@@ -0,0 +1,32 @@
+import { html, LitElement } from 'lit'
```

Comment:
To avoid confusion with ui package, let's not use `wui-` prefix, nor any imports from web3modal/ui. Storybooks components should be their own, completely detached instances. Similarly how `laboratory` will have its own things as well.

Better name like `gallery-placeholder` could be more useful.